darwin/importer/importer.py

```diff
@@ -147,6 +147,7 @@ def maybe_console(*args: Union[str, int, float]) -> None:
     else:
         maybe_console("Using single CPU")
         parsed_files = list(map(importer, tqdm(files) if is_console else files))
+    parsed_files = [f for f in parsed_files if f is not None]
```

Comment:
At this point, `parsed_files` will contain a `None` for every file that the relevant importer was incapable of parsing. Remove it from the list before proceeding to avoid the issue targeted by this change